jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -1340,6 +1365,18 @@ public Method getDeclaredMethod(String name, Class<?>... parameterTypes) throws
  */
 private native boolean isClassADeclaredClass(Class<?> aClass);
 
+/**
+ * Returns true if the class passed in to the method is an enclosed class of
+ * this class, which include both the declared classes and the classes defined
+ * inside a method of this class.
+ *
+ * @param		aClass		The class to validate
+ * @return		true if aClass an enclosed class of this class
+ * 				false otherwise.
+ *
+ */
+private native boolean isClassAEnclosedClass(Class<?> aClass);
```

Comment:
This should be named isClassAnEnclosedClass i.e. An instead of A

Reply:
Replaced with `isClassAnEnclosedClass` everywhere.